src/web/layouts/StandardLayout.tsx

```diff
@@ -507,9 +507,9 @@ export const StandardLayout = ({ CAPI, NAV, format, palette }: Props) => {
 								webTitle={CAPI.webTitle}
 								author={CAPI.author}
 								tags={CAPI.tags}
-								primaryDateline={CAPI.blocks[0].primaryDateLine}
+								primaryDateline={CAPI.webPublicationDateDisplay}
 								secondaryDateline={
-									CAPI.blocks[0].secondaryDateLine
+									CAPI.webPublicationSecondaryDateDisplay
```

Comment:
I suppose we need to make this change for all the other layouts as well? E.g. Immersive

Reply:
Done